ios/FluentUI/Vnext/List/List.swift

```diff
@@ -153,6 +198,15 @@ extension MSFListView {
                 }
             })
             .buttonStyle(ListCellButtonStyle(tokens: tokens, layoutType: layoutType))
+            if let children = cell.children, isExpanded == true {
+                ForEach(children, id: \.self) { child in
+                    MSFListCellView(cell: child,
```

Comment:
I think the MSFListCellView should have a reference to its object data (MSFListVnextCellData) to be able to update its layout when any of its properties change.